auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -160,6 +161,11 @@ public void setUserAgent(String userAgent) {
         factory.setUserAgent(userAgent);
     }
 
+    @SuppressWarnings("unused")
+    public void enableTls12OnPreLollipop() {
```

Comment:
If this is made package private, the users have no means to enable TLS 1.2 on the client, since they need to call this method explicitly to enable support. Do you want to move this into the constructor?